src/js/plugins/plugin.search.js

```diff
@@ -307,20 +229,7 @@ BookReader.prototype.BRSearchCallbackErrorDesktop = function(results, options) {
  * @param {SearchInsideResults} results
```

Comment:
Use this type param in SearchView wherever results are used

Reply:
That's going to translate even though there's no definition of `SearchInsideResults` in this file?